### PR TITLE
[PackageLoading] Remove support for loading empty Package.swift

### PR DESCRIPTION
**Pull request by aciidb0mb3r**
I couldn't find any reference in documentation...
Bug: https://bugs.swift.org/browse/SR-1839

### CODE REVIEW

**Comment by ddunbar** [comment]
@swift-ci please test

File: Sources/PackageLoading/Manifest+parse.swift [hunk 1]
```diff
@@ -40,18 +42,15 @@ extension Manifest {
...
-            let package = PackageDescription.Package.fromTOML(toml, baseURL: baseURL)
-            let products = PackageDescription.Product.fromTOML(toml)
-
-            self.init(path: path, package: package, products: products)
-        } else {
-            // As a special case, we accept an empty file as an unnamed package.
-            //
-            // FIXME: We should deprecate this, now that we have the `init` functionality.
-            self.init(path: path, package: PackageDescription.Package(), products: [])
+        guard let tomlString = try parse(path: path, swiftc: swiftc, libdir: libdir) else {
```
Comment (abertelrud) [inline Sources/PackageLoading/Manifest+parse.swift]: 👍 I appreciate the change to not reuse the name `toml` for both the string representation and the parsed form.

**Comment by abertelrud** [comment]
Diffs look good.

**Comment by ddunbar** [comment]
@swift-ci please test

**Comment by aciidb0mb3r** [comment]
This ok to merge ?

**Comment by ddunbar** [comment]
Go for it!

**Pull request merged**